website/docs/components/table/partials/code/how-to-use.md

```diff
@@ -555,10 +595,10 @@ Here’s a Table implementation that uses an array hash with localized strings f
 <Hds::Table
   @model={{this.model.myDemoData}}
   @columns={{array
-      (hash key='artist' label=(t 'components.table.headers.artist') isSortable=true)
-      (hash key='album' label=(t 'components.table.headers.album') isSortable=true)
-      (hash key='year' label=(t 'components.table.headers.year') isSortable=true)
-      (hash key='other' label=(t 'global.titles.other'))
+      (hash key="artist" label=(t "components.table.headers.artist") isSortable=true)
+      (hash key="album" label=(t "components.table.headers.album") isSortable=true)
+      (hash key="year" label=(t "components.table.headers.year") isSortable=true)
+      (hash key="other" label=(t "global.titles.other"))
```

Comment:
For me, since the array is JavaScript-like non-html code I would use single quotes.Is our standard preference to use double quotes in this context though?

Reply:
> Is our standard preference to use double quotes in this context though?

@KristinLBradley yes (see all the other examples in the `how-to-use.md` files). The reason is that using single quotes is not valid in that context, and triggers an error:

<img width="1791" alt="screenshot_3296" src="https://github.com/hashicorp/design-system/assets/686239/1bad0cbb-25ad-42c1-94ca-3e9dd4068417">